src/main/java/core/basesyntax/Figure.java

```diff
@@ -0,0 +1,13 @@
+package core.basesyntax;
+
+public abstract class Figure implements AreaCalculator, Draw {
+    private final String color;
```

Comment:
remove `final` modifier, this is regular class field.
let's use existing `enum Color` here instead of `String` type